db/migrate/20160406142950_fix_versions_item_type_for_steps.rb

```diff
@@ -0,0 +1,9 @@
+class FixVersionsItemTypeForSteps < ActiveRecord::Migration
+  def up
+    execute "UPDATE versions SET item_type = 'Steps::Approval' WHERE item_type = 'Approval'"
+  end
+
+  def down
+    execute "UPDATE versions SET item_type = 'Approval' WHERE item_type = 'Steps::Approval'"
```

Comment:
@yozlet I changed the `down` to be identical to the `up` -- we're only fixing data, not the schema itself, and we don't ever want to roll this change back.